src/parser/jobs/drk/modules/ResourceSimulator.js

```diff
@@ -396,239 +264,59 @@ export default class Resources extends Module {
-				</Fragment>,
-				severity: SEVERITY.MAJOR,
-				why: <Fragment>
-					Mana was logged at 100% when the fight started, which implies that a <ActionLink {...ACTIONS.DARK_ARTS}/> was not used pre-pull.  Ignore this if mana was at 100% for some other reason.
-				</Fragment>,
-			}))
-		}
 		if (this._droppedTBNs > 0) {
-			this.suggestions.add(new Suggestion({
+			this.suggestions.add(new TieredSuggestion({
 				icon: ACTIONS.THE_BLACKEST_NIGHT.icon,
 				content: <Fragment>
-					One or more <ActionLink {...ACTIONS.THE_BLACKEST_NIGHT}/> applications did not pop, and thus did not generate blood.  This equates to throwing away the mana you could have spent on a <ActionLink {...ACTIONS.DARK_ARTS}/>.
-				</Fragment>,
-				severity: SEVERITY.MAJOR,
-				why: <Fragment>
-					You missed out on {this._droppedTBNs * BLOODSPILLER_BLOOD_COST} blood ({BLOODSPILLER_BLOOD_POTENCY} potency) or {this._droppedTBNs * DARK_ARTS_MANA_POTENCY} potency of Dark Arts buffs.
-				</Fragment>,
-			}))
-		}
-		if (this._noDACarve > 0) {
-			this.suggestions.add(new Suggestion({
-				icon: ACTIONS.CARVE_AND_SPIT.icon,
-				content: <Fragment>
-					One or more <ActionLink {...ACTIONS.CARVE_AND_SPIT}/> was used without <ActionLink {...ACTIONS.DARK_ARTS}/>
-					.  Even though this generates the same amount of mana as <ActionLink {...ACTIONS.SYPHON_STRIKE}/>, it loses out on {DARK_ARTS_CARVE_BOOST_POTENCY} potency, a massive part of your damage as a DRK.
+					One or more <ActionLink {...ACTIONS.THE_BLACKEST_NIGHT}/> applications did not fully use the shield, and thus did not generate a Dark Arts proc to allow free use of <ActionLink {...ACTIONS.EDGE_OF_SHADOW}/> or <ActionLink {...ACTIONS.FLOOD_OF_SHADOW}/>.
 				</Fragment>,
-				severity: SEVERITY.MAJOR,
+				tiers: SEVERITY_THE_BLACKEST_NIGHT,
+				value: this._droppedTBNs,
 				why: <Fragment>
-					You missed out on {this._noDACarve * DARK_ARTS_CARVE_BOOST_POTENCY} potency due to {this._noDACarve} unbuffed carves.
+					You missed out on {this._droppedTBNs} uses of <ActionLink {...ACTIONS.EDGE_OF_SHADOW}/> or <ActionLink {...ACTIONS.FLOOD_OF_SHADOW}/> due to using <ActionLink {...ACTIONS.THE_BLACKEST_NIGHT}/> without the shield being fully consumed.
 				</Fragment>,
 			}))
 		}
-		if (this._wastedBlood > BLOODSPILLER_BLOOD_COST) {
-			this.suggestions.add(new Suggestion({
+		if (this._wastedBlood >= BLOODSPILLER_BLOOD_COST) {
+			const wastedBloodActions = Math.floor(this._wastedBlood / BLOODSPILLER_BLOOD_COST)
+			this.suggestions.add(new TieredSuggestion({
 				icon: ACTIONS.BLOODSPILLER.icon,
 				content: <Fragment>
-					You wasted blood, and could have gotten more uses
-					of <ActionLink {...ACTIONS.BLOODSPILLER}/> or other spenders during the fight
-					(minimum {BLOODSPILLER_BLOOD_POTENCY} gained average combo potency increase each.)
+					You wasted blood, and could have gotten more uses of <ActionLink {...ACTIONS.BLOODSPILLER}/> or other spenders during the fight
```

Comment:
To be fair, I just re-used the pre-existing copy, and cleaned it up a bit ;)

I put in alternate wording... not sure I helped the brevity of the suggestions any, since they're more passive voice-y now, but let me know if you think that's a better direction or if you have an alternative suggestion.

Reply:
Waaaaay better now, thanks for that!